pandas/tests/series/test_dtypes.py

```diff
@@ -149,6 +149,7 @@ def test_astype_str_map(self, dtype, series):
         # see gh-4405
         result = series.astype(dtype)
         expected = series.map(compat.text_type)
+        expected = expected.replace('nan', np.nan)  # see gh-20377
```

Comment:
remove this, this is equiv to ``.astype(str)`` its mapping to a string and is correct